mito-ai/mito_ai/tests/providers_test.py

```diff
@@ -0,0 +1,167 @@
+import os
+from datetime import datetime
+from unittest.mock import patch
+from mito_ai.providers import OpenAIProvider
+
+CHAT_LIMIT = 500
```

Comment:
Instead of recreating this, let's just import `OPEN_SOURCE_AI_COMPLETIONS_LIMIT` so we're making sure to use the correct number.